### ISSUE TITLE
Loading the Bios Contract

### ISSUE THREAD

**Issue by raymondfx** [EOSIO/eos#4494]
I am getting this error 

 ```
cleos set contract eosio build/contracts/eosio.bios -p eosio
Reading WAST/WASM from build/contracts/eosio.bios/eosio.bios.wast...
2859313ms thread-0   main.cpp:2712                 main                 ] Failed with error: Assert Exception (10)
!wast.empty(): no wast file found build/contracts/eosio.bios/eosio.bios.wast
```

**Comment by raymondfx** [comment]
#4490

**Comment by andriantolie** [comment]
Do you have the file `build/contracts/eosio.bios/eosio.bios.wast`?
And also is that cleos the one that you installed locally or the cleos installed inside your docker container? Please share the result of running `type cleos` here. Thanks

**Comment by raymondfx** [comment]
Yes, I have the file `build/contracts/eosio.bios/eosio.bios.wast` and also its the Cleos I installed locally. 
Eosio should be clear with their updates and Documentations, most of the commands at the Moment don't work. There is lack of consistency with the Development updates.

**Comment by andriantolie** [comment]
Hmm, that's strange. I don't have any problem at all running the above command from my eos root folder. Which version are you using? Can you try to share `type cleos` and `ls -al build/contracts/eosio.bios`?

The documentation in developers.eos.io is tied to current stable version v1.0.0. If you find any discrepancy in the documentation, please send an update suggestion on developer.eos.io. Thanks!

**Comment by eosbet-io** [comment]
```
ls -a build/contracts/eosio.bios
``` 

gives

```
.			Makefile		eosio.bios.bc		eosio.bios.wast
..			cmake_install.cmake	eosio.bios.cpp.bc	eosio.bios.wast.hpp
CMakeFiles		eosio.bios.abi		eosio.bios.s
CTestTestfile.cmake	eosio.bios.abi.hpp	eosio.bios.wasm
```

That eosio.bios.wast file exists, but I get that same error.

**Comment by andriantolie** [comment]
If you are you using docker than that makes sense because it is looking for that files inside your docker which won't exist unless you mount the folder. Try to mount the folder in the host to docker container and it should work :)

**Comment by DexHunter** [comment]
Hi! I also used docker, but I found the file `eosio.bios.wast` under `/contracts/eosio.bios` at the docker eosio. I wonder isn't it enough for executing `cleos set contract eosio build/contracts/eosio.bios -p eosio@active`? Thanks!

**Comment by NorseGaud** [comment]
My issue is that it's looking for the file under `/Users/norsegaud/eos/build.release/contracts` and it's actually in
`/Users/norsegaud/eos/build/contracts`

**Comment by jgiszczak** [comment]
@NorseGaud Which document says to use `~/eos/build.release/contracts`?

**Comment by NorseGaud** [comment]
<img width="838" alt="screen shot 2018-08-24 at 3 32 53 pm" src="https://user-images.githubusercontent.com/5896030/44604171-08988d00-a7b3-11e8-9870-4c8d3abfd471.png">

**Comment by harshabing** [comment]
> Yes, I have the file `build/contracts/eosio.bios/eosio.bios.wast` and also its the Cleos I installed locally.
> Eosio should be clear with their updates and Documentations, most of the commands at the Moment don't work. There is lack of consistency with the Development updates.

where is your wasm and wast files in eosio.bios? can you send them to me somehow. i can't find anywhere online. tried reinstalling all but still no files.

**Comment by voorash** [comment]
The documentation has an issue in that it suggests you use the docket install but when you do the command should be:

cleos set contract eosio contracts/eosio.bios -p eosio@active

because inside the docker image the compiled contract files are in the contracts folder in the root directory and there is no build directory with the docker image

**Comment by sugalivijaychari** [comment]
I am also having the same issue.
Error while setting the contract : thread-0 main.cpp:3896 main ] Failed with error: unspecified (0) Unable to resolve path.
On typing the command: cleos set contract ballot /Desktop/IITH_Internship/contracts/Ballot
In that directory, I have all ballot.abi ballot.hpp ballot.cpp and ballot.wasm
